docs/about.md

```diff
@@ -6,39 +6,135 @@
+    <td align="center"><a href="https://github.com/samuelfangjw"><img src="https://avatars.githubusercontent.com/u/60355570?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Samuel Fang</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=samuelfangjw" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/thomastanck"><img src="https://avatars.githubusercontent.com/u/829070?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Thomas Tan</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=thomastanck" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/vig42"><img src="https://avatars.githubusercontent.com/u/42461145?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Vighnesh Iyer</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=vig42" title="Code">💻</a></td>
+    <td align="center"><a href="https://resumednh.netlify.app/"><img src="https://avatars.githubusercontent.com/u/53283766?v=4?s=100" width="100px;" alt=""/><br /><sub><b>dao ngoc hieu</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=daongochieu2810" title="Code">💻</a></td>
+    <td align="center"><a href="https://bandism.net/"><img src="https://avatars.githubusercontent.com/u/22633385?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Ikko Ashimine</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=eltociear" title="Documentation">📖</a></td>
+    <td align="center"><a href="http://www.jovyntan.com"><img src="https://avatars.githubusercontent.com/u/61113575?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jovyn Tan</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=jovyntls" title="Code">💻</a> <a href="https://github.com/MarkBind/markbind/commits?author=jovyntls" title="Documentation">📖</a></td>
+  </tr>
+  <tr>
+    <td align="center"><a href="https://github.com/kaixin-hc"><img src="https://avatars.githubusercontent.com/u/59471320?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Hannah</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=kaixin-hc" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/NaincyKumariKnoldus"><img src="https://avatars.githubusercontent.com/u/87004609?v=4?s=100" width="100px;" alt=""/><br /><sub><b>NaincyKumariKnoldus</b></sub></a><br /><a href="https://github.com/MarkBind/markbind/commits?author=NaincyKumariKnoldus" title="Documentation">📖</a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+</div>
+
+<panel header="Past Record" type="seamless">
```

Comment:
The panel doesn't seem to be working on the preview site?

Reply:
Hi @jonahtanjz, thanks for the heads-up. Interestingly, this is an unexpected hydration issue, in which the all-contributors-cli generates a table without `<tbody>` element, which syntactically is fine but would error out for Vue due to the mismatch in HTML content. I have followed up the issue in the source repo [here](https://github.com/all-contributors/cli/pull/307#issuecomment-1115835646), will wait and see if there are any updates.

In the meantime, it is possible to manually add `<tbody>` into the HTML generated by all-contributors, but it seems like it will be erased if any further update is made automatically, which defeats the purpose of using the bot.

So at the moment, I hack a solution by moving the past record above the contributor table so that the hydration issue will not affect it. See https://deploy-preview-1919--markbind-master.netlify.app/about.html

Reply:
> * whether this is enough or should we present the core teams separately
> * whether we should use more emojis to cover concepts of project-lead, senior-dev, core team

1. I think the current design works (and is less effort to maintain), unless there's some automation.
2. I think adding more emojis would be nice, separating the roles can help new users identify who to contact if they need help.

Reply:
> whether this is enough or should we present the core teams separately

Agree its enough as well.

> whether we should use more emojis to cover concepts of project-lead, senior-dev, core team

On the same lines, I think this is fine too (unless it can be automated). Also as I can't find suitable emojis here to represent these roles: https://allcontributors.org/docs/en/emoji-key. 👀

We can preserve the current formatting (just text, take it out of the panel) for these roles (which should be limited in size) to make maintaining it easier.

> whether to keep the past records or remove entirely

My 2 cents:
- extract and preserve the pm / senior dev list out of the panel per the above problem. (no need for the all-contributors icon). something simple like this https://chromium.googlesource.com/external/github.com/grpc/grpc-go/+/refs/heads/master/MAINTAINERS.md.

  maybe these sections (expand / remove the panel):
  - Current team - pms + senior devs
  - Emeritus - though not sure if we want to move senior / past devs to this list as well as it would be already quite large retroactively
      - ps: we also don't currently have "senior dev" https://markbind.org/about.html, think we could check with @damithc especially on the intention for this. Including the current ones at least would definitely be nice imo.
- use all-contributors for all the other ones to reduce the maintainence work + duplication.

Reply:
> - whether we should use more emojis to cover concepts of project-lead, senior-dev, core team

No preference, same as Ze Yu - can't find a suitable emoji from all-contributors to represent these 

> - whether this is enough or should we present the core teams separately

I'm concerned about the maintainability of the "current team" - do we define it as all current CS3281/2 students and mentors, or all people currently actively contributing to the project? It seems that we'll have to manually maintain this section specifically, which might get forgotten if we happen to have independent contributors not part of CS3281/2. 

> - whether to keep the past records or remove entirely

I think we should keep past project leads/mentors (and maybe maintainers) as it could be interesting information for new devs who wish to know more about MarkBind's project history. I feel there's no harm removing past contributors from the panel since they would already be listed under the "Contributors" section (from all-contributors) below.

Reply:
> I'm concerned about the maintainability of the "current team" - do we define it as all current CS3281/2 students and mentors, or all people currently actively contributing to the project? It seems that we'll have to manually maintain this section specifically, which might get forgotten if we happen to have independent contributors not part of CS3281/2.

Should be everyone who is actively engaged in the project.


> I think we should keep past project leads/mentors (and maybe maintainers) as it could be interesting information for new devs who wish to know more about MarkBind's project history. I feel there's no harm removing past contributors from the panel since they would already be listed under the "Contributors" section (from all-contributors) below.

Yes, it's better to keep if we can.